packages/client-search/src/types/Log.ts

```diff
@@ -91,6 +91,6 @@ export type Log = {
     /**
      * The user token of the query.
      */
-    readonly userToken: string;
```

Comment:
userToken is optional

Reply:
You mean `user_token` right?

Reply:
Fixed, can you validate the pull request please?

Reply:
Yes, `user_token` sorry.